src/app/workspace/resource/representation/add-region-form/add-region-form.component.html

```diff
@@ -0,0 +1,42 @@
+<form [formGroup]="regionForm">
+    <mat-form-field>
+        <mat-label>Comment</mat-label>
+        <input matInput formControlName="comment">
```

Comment:
Comment field should be a textarea. But I think the form style could also be done in a separate, style refactoring PR.

Reply:
I changed it to a textarea for now, but I agree that a style refactoring PR will be needed.